### DISCUSSION TITLE
Synchronize folder structure

### DISCUSSION THREAD

**Discussion by notrub32** [WinMerge/winmerge#2040] [Q&A]
If I have a folder structure, COPYING updates from one side to another seems to only be capable of copying everything - not just the changed files, even though if I navigate into the hierarchy, it correctly detects which files are changed and which are not.

I can work around this by navigating to each folder in turn and performing a merge, but this makes the "include subfolder" option pointless.

TLDR - what I'd like to do is select a top level folder, select differences and just copy over the changed files in all subfolders. Is this not possible?

**Comment by sdottaka** [comment]
Unfortunately, current WinMerge doesn't allow you to do that.
